sydent/http/servlets/__init__.py

```diff
@@ -18,6 +18,14 @@
 import copy
 
 
+class MatrixRestError(Exception):
```

Comment:
Might be worth adding a quick docstring to say that this gets handled by `jsonwrap` specially